Runar/Runar/Application/SceneDelegate.swift

```diff
@@ -31,6 +31,8 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     func sceneDidBecomeActive(_ scene: UIScene) {
         // Called when the scene has moved from an inactive state to an active state.
         // Use this method to restart any tasks that were paused (or not yet started) when the scene was inactive.
+        
+        UIApplication.shared.applicationIconBadgeNumber = 0
```

Comment:
Why do you set badgeNumber directly?

Reply:
Setting badgeNumber is needed here so that when opening/resuming the application, the badgeNumber is reset.

Reply:
Ok